templates/core/services/thank-you.html

```diff
@@ -2,49 +2,50 @@
 
 {% block title %}Thank you | AWS{% endblock %}
 
-{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+{% block head_extra %}<meta name="robots" content="noindex" />{% endblock %}
 
 {% block content %}
-<div class="p-strip is-bordered is-deep">
-  <div class="row">
-    <div class="col-7">
-      <h1>Thank you for your interest in SMART START</h1>
-      <p class="p-heading--4">A member of our team will be in touch within one working day.</p>
-    </div>
-    <div class="col-5 u-vertically-center u-hide--medium u-hide--small u-align--center">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/52d53696-picto-thankyou-midaubergine.svg",
-          alt="",
-          width="200",
-          height="200",
-          hi_def=True,
-          loading="auto"
-        ) | safe
-      }}
+  <div class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="u-no-margin--bottom">Thank you for your interest in SMART START.</h1>
+        <h2>A member of our team will be in touch within one working day.</h2>
+      </div>
+      <div class="col">
+        <div class="p-image-container u-hide--small">
+          <img class="p-image-container__image p-strip is-shallow"
```

Comment:
The addition of the `p-strip` maligns this image like so:
![image](https://github.com/user-attachments/assets/f4896b19-9f09-4aeb-8480-ef701854ef4f)

The following provides the correct spacing and also matches the recommended usage of this component more closely, see [here](https://vanillaframework.io/docs/patterns/images) 
```suggestion
        <div class="p-image-container is-cover u-hide--small">
          <img class="p-image-container__image"
```